### ISSUE TITLE
Set date & time service 

### ISSUE THREAD

**Issue by Igor-Yavych** [signalkraft/mypyllant-component#147]
### Describe the feature you want
Hello, I've been using [this](https://github.com/thomasgermain/vaillant-component) component but alas, it's no longer working due to migration to myvaillant. One very useful service it had, that is missing from this component, is one to set time date in the system. It was superbly useful, considering frequent blackouts in Ukraine (I had an automation in HA to set it to current time on HA launch). Would it be possible to add such a service to this component?

**Comment by signalkraft** [comment]
Unfortunately I never found a way to set this in the app or over the API. If you discover a way to update date & time, send some screenshots please!

**Comment by Igor-Yavych** [comment]
@boelle no, but it seems there is no need for that as new app (myVAILLANT) or the new firmware that was installed when the switch to the new app happened handle this part on its own as I haven't had to set date manually even once since then
